### ISSUE TITLE
Firefox graphmenu issue

### ISSUE THREAD

**Issue by reboot1983** [Cacti/cacti#667]
Please see: http://forums.cacti.net/viewtopic.php?f=2&t=57409&p=267119

Strange little thing in the latest Firefox.
I was able to reproduce in the beginning, but now it just works..
I've installed the latest Firefox x64 for this on my Windows 10 machine. First view of the graph tab shows exactly like Rno explains. Then I made the 'mistake' of clicking on the first item that was visible (the line of 1px that's visible). Then the browser reloaded the page with the right URL parameters and I was never again able to reproduce this issue again.

**Comment by cigamit** [comment]
There were some changes here in a vain attempt to 'resolve' issues with the Chrome browser not returning some dimensional values.  Chrome, though it is fast is a massive pain in the arse.  Many things are just broken with rendering pages.  I went to the open chromium bug tracker and almost lost it when I saw the number of rendering issues.  Plus it holds onto the cached CSS and JS until who knows when.  Neither a refresh or CTRL-R do it.
